src/WebJobs.Script.WebHost/Diagnostics/DiagnosticLogger.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Azure.WebJobs.Logging;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics
+{
+    public class DiagnosticLogger : ILogger
```

Comment:
Should this be named something like AzureMonitorDiagnosticLogger? Diagnostic is pretty general.

Reply:
Good idea.